src/mono/wasm/Wasm.Build.Tests/BuildTestBase.cs

```diff
@@ -342,6 +342,8 @@ protected static BuildArgs ExpandBuildArgs(BuildArgs buildArgs, string extraProp
             if (UseWebcil) {
                 extraProperties += "<WasmEnableWebcil>true</WasmEnableWebcil>\n";
             }
+            if (buildArgs.WasmIncludeFullIcuData)
```

Comment:
The test class needs to be listed in `eng/testing/scenarios/BuildWasmAppsJobsList.txt`. I'll open a separate PR to generate that list automatically, but for now the name is needed in there.